### ISSUE TITLE
Accessibility of video tabs

### ISSUE THREAD

**Issue by ibrahimjaved12** [mitodl/ocw-hugo-themes#1219]
## Accessibility of:
1.  `base-theme/layouts/partials/video_tab_title.html`
2.  `base-theme/layouts/partials/video_expandable_tab.html`
Code:
```
<i class="material-icons toggle md-18"></i>
<span class="tab-title">
  {{- .tabTitle -}}
</span>

```
Go to link: https://ocw.mit.edu/courses/3-091-introduction-to-solid-state-chemistry-fall-2018/resources/lecture-2/
And use a screen reader on "Transcript" and the video download icon.

They seem to be redundant. Like, each word is read multiple times for both of the buttons. I think this might be linked to `base-theme/layouts/partials/video_expandable_tab.html` as we have the relevant `aria-label="Download Video and Transcript"` there. Investigate both of these files. Also check for other tab titles: "Related Resources", "Lecture Notes", etc how a screen reader reads them.

## Screenshot
<img width="571" alt="image" src="https://github.com/mitodl/ocw-hugo-themes/assets/109785089/8307c6b5-49b5-47e2-b748-799f29185d3b">

**Comment by ChristopherChudzicki** [comment]
For some context, I want to share a little bit more why this is happening and what I think might be a better layout. Basically, it's because we are nesting interactive elements (multiple buttons inside a tab) and I think we should probably not do that.

Below, I have annotated the aria [roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles). These are either set explicitly via `role=""`, or they are they are implicit, such as `<a />` elements implicitly having role "link".

<img width="1176" alt="Screenshot 2023-08-28 at 11 21 41 AM" src="https://github.com/mitodl/ocw-hugo-themes/assets/9010790/02e31a95-9d48-49ac-b02f-2e1dcf66a49d">

Note that "button", "link", and "tab" are interactive roles.

The screenreader is reading the elements' [accessible names](https://www.w3.org/WAI/ARIA/apg/practices/names-and-descriptions/). Every element can be  explicitly given an accessible name via `aria-label="..."`. Elements whose **roles are interactive** often compute their accessible name from their content..

So: The "tab" "tab" is interactive; it computes its name from its content,  button (1) with name "Transcript", and button (2) with name "Download Video and Transcript" (set via `aria-label`). The final name is "Transcript Download Video and Transcript".  Then screenreaders read:
1. The tab's name: "Transcript Download Video and Transcript"
2. But the content is also interactive, so it reads those, too.

### Suggestion
IMO, the best solution here would be to not put buttons *inside* the tab. The tab itself should be a button:

<img width="1212" alt="Screenshot 2023-08-28 at 11 21 48 AM" src="https://github.com/mitodl/ocw-hugo-themes/assets/9010790/444fd0ba-dc29-4d09-b986-dccf920790d7">

- no more nested interactive elements ==> accessible name computation is easier to understand
- Screenreader would read "Transcript" for the tab, and whatever our aria-label is for the button. *Currently 'Download Video and Transcript', though IMO "Show Downloads" would be better.*
- Increases size of download menu toggle => easier for mobile
